app/code/Magento/WishlistGraphQl/Model/Resolver/WishlistRemoveItemResolver.php

```diff
@@ -0,0 +1,152 @@
+     * @param WishlistItemCollectionFactory $wishlistItemCollectionFactory
+     * @param StoreManagerInterface $storeManager
+     */
+    public function __construct(
+        WishlistResourceModel $wishlistResource,
+        WishlistItemResourceModel $wishlistItemResource,
+        WishlistFactory $wishlistFactory,
+        WishlistItemCollectionFactory $wishlistItemCollectionFactory,
+        StoreManagerInterface $storeManager
+    ) {
+        $this->wishlistResource = $wishlistResource;
+        $this->wishlistItemResource = $wishlistItemResource;
+        $this->wishlistFactory = $wishlistFactory;
+        $this->wishlistItemCollectionFactory = $wishlistItemCollectionFactory;
+        $this->storeManager = $storeManager;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        $wishlist = $this->getWishlistById($args['input']['wishlist_id']);
+        $wishlistId = $wishlist->getId();
+
+        $this->deleteWishlistItems($wishlist, $args['input']['wishlist_items_ids']);
+
+        return [
+            'wishlist' => [
+                'id' => $wishlistId,
+                'sharing_code' => $wishlist->getSharingCode(),
+                'updated_at' => $wishlist->getUpdatedAt(),
+                'items_count' => $wishlist->getItemsCount(),
+                'name' => $wishlist->getName(),
+                'model' => $wishlist,
+            ]
+        ];
+    }
+
+    /**
+     * Loads Wishlist by the wishlist id
+     *
+     * @param int $wishlistId
+     * @return Wishlist
+     */
+    private function getWishlistById($wishlistId): Wishlist
+    {
+        /** @var Wishlist $wishlist */
+        $wishlist = $this->wishlistFactory->create();
+        $this->wishlistResource->load($wishlist, $wishlistId);
+        return $wishlist;
+    }
+
+    /**
+     * @param Wishlist $wishlist
+     * @param array $ids
+     * @throws LocalizedException
+     */
+    private function deleteWishlistItems(Wishlist $wishlist, Array $ids)
+    {
+        $wishlistItems = $this->getWishListItems($wishlist, $ids);
+
+        foreach ($wishlistItems as $whislistItem) {
+            try {
+                $this->wishlistItemResource->delete($whislistItem);
+            } catch (\Exception $e) {
+                throw new LocalizedException(__('Failed to delete the wishlist item'));
+            }
+        }
+    }
+
+    /**
+     * @param Wishlist $wishlist
+     * @param array $ids
+     * @return array
+     */
+    private function getWishListItems(Wishlist $wishlist, Array $ids): array
+    {
+        /** @var WishlistItemCollection $wishlistItemCollection */
+        $wishlistItemCollection = $this->wishlistItemCollectionFactory->create();
+        $wishlistItemCollection
+            ->addWishlistFilter($wishlist)
+            ->addStoreFilter(array_map(function (StoreInterface $store) {
```

Comment:
We should get `store` from $context.
`$storeId = (int)$context->getExtensionAttributes()->getStore()->getId();`